src/main/java/frc/robot/commands/FullAcquireSequenceCommand.java

```diff
@@ -0,0 +1,52 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Storage;
+import frc.robot.subsystems.Acquisition;
+
+public class FullAcquireSequenceCommand extends Command {
+    private Storage storage = Storage.getInstance();
+    private Acquisition acquisition = Acquisition.getInstance();
+    private boolean isFinished;
+
+    public FullAcquireSequenceCommand() {
+        addRequirements(storage, acquisition);
+    }
+
+    @Override
+    public void initialize() {
+
+    }
+
+    @Override
+    public void execute() {
+        isFinished = false;
+        acquisition.acquire();
+
+        if (acquisition.isNotePresent()) {
+            storage.runTransition();
+            acquisition.reverseMotors();
+
+            if (storage.noteEnteringStorage()) {
+                storage.stopTransition();
+                storage.runStorage();
+
+                if (storage.noteExitingStorage()) {
+                    isFinished = true;
+                }
+            }
+
+        }
+    }
+
+    @Override
+    public boolean isFinished() {
```

Comment:
Only important note is in the case where the command doesn't succeed at any point it doesn't have a time limit, my only real concern is there is no default timer.

Reply:
The command should stop if you stop pushing the button since we would probably use a whileTrue for this

Reply:
Got it